Rb.Forms.Barcode.Droid/BarcodeScannerRenderer.cs

```diff
@@ -2,104 +2,120 @@
 using System.ComponentModel;
 using Xamarin.Forms;
 using Xamarin.Forms.Platform.Android;
-
 using Rb.Forms.Barcode.Pcl;
 using Rb.Forms.Barcode.Pcl.Logger;
 using Rb.Forms.Barcode.Droid;
 using Rb.Forms.Barcode.Droid.Camera;
-using Rb.Forms.Barcode.Droid.Decoder;
 using Rb.Forms.Barcode.Droid.Logger;
-
 using Android.Views;
 using System.Threading.Tasks;
 using Rb.Forms.Barcode.Droid.View;
+using Android.Content.PM;
+using Android.Runtime;
```

Comment:
have that turned on, will check it out :) But the chinese could be a shop stopper for me :p